components/server/src/routes/report.py

```diff
@@ -111,7 +112,7 @@ def export_report_as_pdf(report_uuid: ReportId):
     return response.content
 
 
-@bottle.get("/api/v3/report/<report_uuid>/json")
+@bottle.get("/api/v3/report/<report_uuid>/json", authentication_required=True)
 def export_report_as_json(database: Database, report_uuid: ReportId):
     """Return the quality-time report, including iencrypted credentials for api access to the sources."""
```

Comment:
Caught an unrelated typo:
```suggestion
    """Return the quality-time report, including encrypted credentials for api access to the sources."""
```